src/constants/memory/hram.asm

```diff
@@ -191,16 +191,18 @@ hVolumeLeft::
 hMusicFadeInTimer::
   ds 1 ; FFAB
 
-; Unlabeled
-hFFAC::
+; Status of the staircase in the current room.
+; (There can be only one staircase per room.)
+; See STAIRCASE_* constants for possible values.
+hStaircase::
   ds 1 ; FFAC
 
-; Unlabeled
-hFFAD::
+; X position of staircase in the current room.
```

Comment:
Might be good to note that this is a pixel position, not a tile position. For example the warp position data is in tile positions, while this is in pixel positions.

Reply:
Indeed ; updated.